### ISSUE TITLE
scrollReveal not working with content loaded via ajax 

### ISSUE THREAD

**Issue by cmaaga** [jlmakes/scrollreveal#52]
Hi,
I try to use scrollReveal in my wordpress website. You can see it here www.wytworniaprojektu.eu (this is develop version). As you cas see I use Advanced Ajax Page Loader plugin. And this is my problem. Scroll Reveal work only once. I reload code of scrollReveal.js (see inside console log) but nothing happen. 

Any idea? 

Sorry for my english.

**Comment by rmuyinda** [comment]
Guys Paddon me, Is it possible to Use ScrollReveal with Ajax Call on independ scrolls and if so I would like to see an simple example with Json

**Comment by julianlloyd** [comment]
@rmuyinda 

If you instantiate scrollReveal like this:
```JavaScript
window.sr = new scrollReveal();
```
you will have access to the `sr` object, so you can call `sr.init()` anytime afterwards to re-scan for new elements. So in other words, if you’re adding new elements to the page with AJAX, all you need to do is trigger `sr.init()` in your AJAX success callback.

**Comment by gian-reto** [comment]
Hello guys. First, thanks for this issue, it really helped me getting things to work with dynamic loading of content after searching for the problem somewhere else for hours! :+1: 
I shill have a problem though – correct me if I'm wrong, but it appears that custom effects defined with the data-sr attribute don't work for loaded content (although the elements I try to add later are exactly the same as the initial ones which work perfectly)... So I tried setting custom values in the options when creating the ScrollReveal instance: ```window.sr = new scrollReveal({ options });``` but still didn't work. My last attempt was adding the options in the ```init();``` call so it looked like this:
```
sr.init({
    enter: 'bottom',
    easing: 'ease-out',
    move: '60px',
    scale: {
      direction: 'up',
      power: '0%' },
    over: '0.5s',
    reset: false
  });

**Comment by icesh** [comment]
I am not sure if it is appropriate to post my question on this post. I've got a question when using this item.
First thanks for Julian for making such a good item! 
May i know in detail how to use this plugin appropriately in a specific WP theme after uploading the "scrollReveal.min" in dist folder to wp/theme/js directory. Sorry that i am not familiar with the code but still do want to give it a try! Appreciate if anyone could help. Thank you very much

**Comment by gian-reto** [comment]
@icesh I'm not an author of this repo but I think everyone would appreciate it if you'd open a new issue, because your question doesn't belong to the current discussion here and would be more helpful to others as standalone. Also, it may be helpful to you to check out the docs: https://github.com/julianlloyd/scrollReveal.js/wiki/Getting-Started

**Comment by icesh** [comment]
@gian-reto Yes. Sorry for that and I have opened a question already. Thanks^^

**Comment by gian-reto** [comment]
@icesh no problem I wish you good luck finding an answer maybe I can help.

**Comment by icesh** [comment]
@gian-reto Thank you! Could you pls refer to the issue that i opened a while ago or how could i leave you message on this website? I am sorry for sort of keeping post something inappropriate here. Thanks again!
https://github.com/julianlloyd/scrollReveal.js/issues/128
